src/main.rs

```diff
@@ -327,7 +328,9 @@ async fn provide_interactive(
 
     let (db, hash) = provider::create_collection(sources).await?;
 
-    println!("Collection: {}\n", Blake3Cid::new(hash));
+    out_writer
+        .println(format!("Collection: {}\n", Blake3Cid::new(hash)))
+        .await;
     for (_, path, size) in db.blobs() {
         println!("- {}: {} bytes", path.display(), size);
```

Comment:
what about this print?

Reply:
!!!!

added to the correct output & tested.